openmmtools/states.py

```diff
@@ -1478,10 +1617,8 @@ class SamplerState(object):
     velocities
     box_vectors : 3x3 simtk.unit.Quantity.
         Current box vectors (length units).
-    potential_energy : simtk.unit.Quantity or None
-        Potential energy of this configuration.
-    kinetic_energy : simtk.unit.Quantity
-        Kinetic energy of this configuration.
+    potential_energy
```

Comment:
What happened to the docstring here?

Reply:
`potential_energy` and `kinetic_energy` are properties now so I moved their docstrings in the property definitions.